.github/workflows/tools.yml

```diff
@@ -123,27 +111,13 @@ jobs:
             label: dependencies
             run: |
               NEW_VERSION=$(gh api repos/libuv/libuv/releases/latest -q '.tag_name|ltrimstr("v")')
```

Comment:
I was unsure how to port this line into the update script. I'll take a deeper look.

If someone has an idea, feel free to share ^

Reply:
I missed that earlier, but FYI you could do something along the lines of

```js
NEW_VERSION="$("$NODE" --input-type=module <<'EOF'
const res = await fetch('https://api.github.com/repos/libuv/libuv/releases/latest');
if (!res.ok) throw new Error(`FetchError: ${res.status} ${res.statusText}`, { cause: res });
const { tag_name } = await res.json();
console.log(tag_name.replace('v', ''));
EOF)"
```

Reply:
Do you want to try to fit that in? Or we can land this as is, and take this to a follow up PR, you've been dragging this one for long enough :)

Reply:
I definitely prefer to try it ^^ It will abstract the logic a bit more into the script which is the first motivation of this PR.

Reply:
done 🎉 @aduh95 let me know WDYT ^^

We have something homogenous now. GH scripts and updater scripts share same responsabilites.

Reply:
Linter is not happy btw. I didn't see any parse error on my machine. I'll take a deeper look.

Reply:
WDYT about this?

<img width="1210" alt="Screenshot 2023-02-24 at 12 38 04" src="https://user-images.githubusercontent.com/22824417/221170068-619f420a-3f01-4689-b093-730343d84971.png">

Reply:
Please consider copy/pasting the text rather than taking screenshots, as your local settings may be hard to decipher for the rest of us (e.g. folks with visual disabilities), and is also more data to download, which is never nice for those on limited data plans 🙂

I don’t think this is ideal, wouldn’t adding a line return before closing the parentheses works, as the linter suggests? Meaning replacing `EOF)` with `EOF\n)`.

Reply:
Sorry, I didn't explain, but the screenshots were to show the syntax highlight that was totally broken with the previous version.

I finally add the put the EOF in the line below and it worked!